Library/Homebrew/cmd/audit.rb

```diff
@@ -234,6 +234,32 @@ def audit_options
     end
   end
 
+  def audit_desc
+    # For now, only check the description when using `--strict`
+    return unless @strict
+
+    desc = formula.desc
+
+    unless desc
+      problem "All formulas should have a desc."
```

Comment:
Thinking about the way we've worded other things, it may be better to drop the `all`. We audit individual formula, so referring to them en mass is a bit of a clash. `Formula should have a desc (Description)` or something.